[41492] Clicking in "include projects" breaks the page

HDinger opened the pull request:
A fixed position prevents an effect on the scrolling container compared to an absolute positioning

https://community.openproject.org/projects/openproject/work_packages/41492/activity

HDinger committed:
A fixed position prevents an effect on the scrolling contianer compared to an absolute positioning (see #41492).

b12f reviewed (changes requested):
I'm pretty sure this breaks keyboard a11y. There might be better ways to hide the checkbox (opacity 0.001 works afaik), but we need it to stay there

HDinger commented:
> I'm pretty sure this breaks keyboard a11y. There might be better ways to hide the checkbox (opacity 0.001 works afaik), but we need it to stay there

Opacity or visibility don't work because the element then lays next to the fake one, decreasing its width. I changed it now to `position: fixed` as it should not influence the scrolling behaviour.
